source/common/tracing/zipkin/zipkin_tracer_impl.cc

```diff
@@ -0,0 +1,194 @@
+#include "common/tracing/zipkin/zipkin_tracer_impl.h"
+
+#include "common/common/enum_to_int.h"
+#include "common/http/headers.h"
+#include "common/http/message_impl.h"
+#include "common/http/utility.h"
+#include "common/tracing/http_tracer_impl.h"
+#include "common/tracing/zipkin/zipkin_core_constants.h"
+
+#include "spdlog/spdlog.h"
+
+namespace Zipkin {
+
+ZipkinSpan::ZipkinSpan(Zipkin::Span& span) : span_(span) {}
```

Comment:
nit: it would be optimal if you remove Zipkin from all of the class names now, since you are inside the Zipkin namespace. You can veto this but I would appreciate it. 😉